File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.js
/**
 * @param {string} allowed
 * @param {string[]} words
 * @return {number}
 */
var countConsistentStrings = function(allowed, words) {
  const set = new Set(allowed)
  const isConsistent = s => {
    for (let ch of s) {
      if (!set.has(ch)) return false
    }
    return true
  }
  let ans = 0
  for (word of words) {
    if (isConsistent(word)) ans++
  }
  return ans
};

File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.go
func countConsistentStrings(allowed string, words []string) int {
  count := 0
  for _, word := range words {
    count ++
    for _, r := range word {
      if !strings.ContainsRune(allowed, r) {
        count --
        break
      }      
    }
  } 
  return count
}